lib/mule.js

```diff
@@ -89,7 +89,7 @@ function addreloader(mule, target) {
 	var rld = $('<div class="button">')
 	rld.text('\u21bb')
 	if (mule.data) {
-		var updated = new Date(mule.data.query.created)
+		var updated = new Date(mule.data.created)
```

Comment:
the `query` object and it's `created` property are inherent to the yql api. With your changes this is same as `new Date()` which is not correct if loading from cache. You need to set this timestamp yourself after updating.